src/resources/api.test.ts

```diff
@@ -57,7 +57,9 @@ describe('Api', () => {
 	["get", "post", "delete", "patch", "put", "options"].forEach(method => {
 		describe(`when creating a new ${method} handler`, () => {
 			beforeAll(async () => {
-				await api("main")[method]("/test/", mockFn, {
+				await api("main1", {
```

Comment:
Renamed due to resource caching issue.

Created APIs above were causing api path not to be respected due to multiple `main` API declaration.

Need to think about the best way to handle base option conflicts.